test/hexpm/accounts/audit_logs_test.exs

```diff
@@ -0,0 +1,66 @@
+defmodule Hexpm.Accounts.AuditLogsTest do
+  use Hexpm.DataCase, async: true
+
+  alias Hexpm.Accounts.AuditLogs
+
+  describe "all_by(user)" do
```

Comment:
Do we need to move these tests to `AuditLogTest`?

I chose to put them here because `AuditLogs.all_by/1` is the contract between controller and context. And I think it's important to cover this contract with tests.

Reply:
This is the right place to put them.